script/component-migration/samples/AlertBox.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 
 export function Sample() {
   const myStatus = 'info';
   const someContent = 'Testing Testing';
   const myHeadline = 'Pay attention!';
 
   return (
-    <AlertBox status={myStatus} headline={myHeadline} content={someContent} />
+    <va-alert status={myStatus} visible>
+      <h3 slot="headline">{myHeadline}</h3>
+      {someContent}
+    </va-alert>
```

Comment:
This 'sample' component was supposed to be used as a file that you could run the component-migration script on, and then it would run the code-mod on it to show an engineer how to migrate from the old alert to the new one. 

I suppose if we are completing the migration altogether on vets-website, then this file could basically just be deleted as it served only as an example.